src/classes/RoomRepo.js

```diff
@@ -2,25 +2,23 @@ import Room from './Room.js';
 
 class RoomRepo {
   constructor(roomsData, bookingsData) {
-    this.rooms = roomsData;
-    this.bookings = bookingsData;
+    this.rooms = roomsData.rooms;
+    this.bookings = bookingsData.bookings;
   }
 
   findAllRoomBookings(bookingsData, patronsData) {
```

Comment:
Totally agree that the collection of Rooms probably keeps track of the bookings among them. Perhaps space to explore if rooms also needs to know about these things or if you can have a single source of truth.